docs/_guide/styles.md

```diff
@@ -4,66 +4,247 @@ title: Styles
+
+Constuctible Stylesheets allow LitElement to parse styles exactly once and reuse the resulting Stylesheet object for maximum efficiency.
+
+### Limitations
+
+Static styles apply to all instances of an element. Any expressions in the style text are evaluated and included once, then reused for all instances. If you need to vary styles per-instance we reccomend using CSS custom properties, or inline styles.
+
+For security reasons, the only types of values that can be interpolated into static styles are values returned by the `cssLiteral` and `unsafeCSS` template tags.
+
+```js
+const mainColor = cssLiteral`red`;
+
+class MyElement extends LitElement {
+  static styles = css`
+    :host {
+      display: block;
+      color: ${mainColor}
+    }
   `;
 }
 ```
 
-{% include project.html folder="docs/style/hostselector" openFile="my-element.js" %}
+## Inline styles
 
-See the MDN documentation on [:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:host), [:host()](https://developer.mozilla.org/en-US/docs/Web/CSS/:host()), and [pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) for more information.
+You can also style a shadow root with inline styles right in your element template. We still reccomend static styles, but in some cases you may want to vary the CSS per-element. One way to do this is with bindings in `<style>` elements. It's important to note that this will not work with shadow DOM polyfills like ShadyCSS.
 
-### Set host element display styles
+```js
+import {LitElement, property} from 'lit-element';
+
+class MyElement extends LitElement {
+  @property() mainColor = 'blue';
+  render() {
+    return html`
+      <style>
+        :host {
+          color: ${this.mainColor};
+        }
+      </style>
+    `;
+  }
+}
+```
 
-Two best practices for working with custom elements are:
+We strongly reccomend static styles, CSS custom properties, or [lit-html's `classMap` or `styleMap` directives](TODO) if you're stying non-host shadow root contents.
 
-* Set a `:host` display style such as `block` or `inline-block` so that your component's `width` and `height` can be set.
+## External stylesheets
 
-* Set a `:host()` display style that respects the `hidden` attribute.
+You can load an external stylesheet into a shadow root with a `<link>` element:
+
+```js
+import {LitElement} from 'lit-element';
+
+class MyElement extends LitElement {
+  render() {
+    return html`
+      <link rel="stylesheet" href="./styles.css">
+    `;
+  }
+}
+```
+
+This can be a good way to load CSS generated from tools like SASS/LESS.
+
+There are some important caveats though:
+
+* External styles can cause a flash-of-unstyled-content (FOUC) while they load.
+* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publically.
```

Comment:
* As we are not documenting how to build apps, (covered in PWA starter kit) should this content be here or somewhere else? @jbingham @graynorton 
* If it belongs here, someone is bound to ask how to get the URL the the element was imported from (can't remember how to do it, import meta something something..) should we cover that somewhere?